.github/workflows/action.yml

```diff
@@ -0,0 +1,61 @@
+name: Robottelo - CI
+
+on:
+  push:
+  pull_request:
+    types: ["opened", "synchronize", "reopened"]
+
+env:
+    PYCURL_SSL_LIBRARY: gnutls
+
+jobs:
+  codechecks:
+    name: 📃 Code Quality
+    runs-on: ubuntu-latest
```

Comment:
Can we change this to a fedora image?

Reply:
Looks like they dont support it out of the box, I saw this thread https://github.com/actions/virtual-environments/issues/1443, Any specific reasons we have to use this with fedora image? because we were using xenial Travis which is ubuntu distribution. I don't think we should be blocking this against fedora image.

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

Not without changing the pipeline to be a docker image strategy. Github offers ubuntu VMs for these pipelines. Same as in travis, we're running on ubuntu.